decidim-participatory_processes/db/migrate/20180125102537_add_reference_to_processes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AddReferenceToProcesses < ActiveRecord::Migration[5.1]
+  class ParticipatoryProcess < ApplicationRecord
+    self.table_name = :decidim_participatory_processes
+  end
```

Comment:
If you want to rely on ActiveRecord `after_commit` hooks, I think you need to either drop this fake model, or copy the code that's needed to update the reference into the fake model, no?

Reply:
Eh you're right, I wanted to remove this and forgot, thanks!